packages/forklift-console-plugin/src/modules/Providers/utils/validators/provider/openshift/openshiftSecretFieldValidator.ts

```diff
@@ -15,8 +15,11 @@ import { validateK8sToken, validatePublicCert, ValidationMsg } from '../../commo
  *
  * Validation msg and description should add information about the validation type.
  */
-export const openshiftSecretFieldValidator = (id: string, value: string): ValidationMsg => {
-  const trimmedValue = value.trim();
+export const openshiftSecretFieldValidator = (
+  id: string,
+  value: string | boolean,
+): ValidationMsg => {
+  const trimmedValue = value?.toString().trim();
```

Comment:
```suggestion
  const trimmedValue = value?.trim();
```
value should only be string, see comment above